Menu/Menu.js

```diff
@@ -1,10 +1,18 @@
-
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+    // Toggle the "menu--open" class on your menu refence.
+    menu.classList.toggle('menu--open');
+    TweenMax.fromTo(".menu", 0.5, {
```

Comment:
Nice job going for stretch on this, this library is a really good one to have under your belt.